plugins/circleci/src/pages/BuildsPage/BuildsPage.tsx

```diff
@@ -19,13 +19,18 @@ import { Grid } from '@material-ui/core';
 import { Builds as BuildsComp } from './lib/Builds';
 import { Layout } from '../../components/Layout';
 import { PluginHeader } from '../../components/PluginHeader';
+import { AppStateProvider } from '../../state/AppState';
+import { Settings } from '../../components/Settings';
 
 const BuildsPage: FC<{}> = () => (
-  <Layout>
-    <Content>
-      <Builds />
-    </Content>
-  </Layout>
+  <AppStateProvider>
```

Comment:
Doesn't this mean that state (e.g. settings,cache) is going to be invalidated between different pages?

Reply:
Fixing.